pom.xml

```diff
@@ -19,8 +19,25 @@
     <java.version>1.8</java.version>
     <maven.compiler.version>1.8</maven.compiler.version>
     <jackson.version>2.11.2</jackson.version>
+    <!-- metadb common centralized config properties -->
+    <cmo_metadb_common.groupId>com.github.ao508</cmo_metadb_common.groupId>
```

Comment:
need to update these to the upstream/master commit and group id once https://github.com/mskcc/cmo-metadb-common/pull/4 is merged